client/index.js

```diff
@@ -21,22 +22,37 @@ function App() {
 
   function getTopMovies(){
     let movieData = [];
-    getTopMoviesApi().then(data => {
-        data.results.forEach((x, i) => movieData.push({"Movie": x.title, "Overview": x.overview, "Release Date": x.release_date, "Language": x.original_language}));
-        setMovies(movieData);
-        localStorage.setItem('topMovies', movieData);
-    })
-    .catch(err => console.log(err));
+    console.log("getting top movies");
+    let cachedMovies = localStorage.getItem('topMovies');
```

Comment:
re-implements cache outside of Api - separates Fetching and whether-to-fetch - I like it 👍